packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -362,20 +363,12 @@ export const EntityHeaderTable = (props: EntityHeaderTableProps) => {
         )}
       </Box>
       {totalRowCount > 0 && (
-        <Box
-          sx={theme => ({
-            overflow: 'auto',
-            maxHeight: '250px',
-            paddingLeft: '2px',
+        <StyledTableContainer
```

Comment:
Use new `StyledTableContainer` component, which provides shared/standard styling for tables